docs/source/notebook-components/weaver_example.ipynb

```diff
@@ -1152,43 +1195,54 @@
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  22% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]   5% running    Loading package content done.\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  25% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]   6% running    Retrieve package inputs done.\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  27% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]   8% running    Convert package inputs done.\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  30% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]   9% running    Checking package prerequisites... (operation could take a while depending on requirements)\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  33% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]   9% running    Package ready for execution.\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  36% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]  10% running    Running package...\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  39% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]  10% running    Preparing to launch package ncdump.\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  42% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump] WPS-1 Package resolved from requirement/hint: WPS1Requirement\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  44% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]  11% running    [provider: https://pavics.ouranos.ca/twitcher/ows/proxy/hummingbird, step: ncdump] - Preparing process for remote execution.\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  47% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]  14% running    [provider: https://pavics.ouranos.ca/twitcher/ows/proxy/hummingbird, step: ncdump] - Process ready for execute remote process.\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  50% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]  18% running    [provider: https://pavics.ouranos.ca/twitcher/ows/proxy/hummingbird, step: ncdump] - Staging inputs for remote execution.\n",
+      "[2022-08-18 20:44:03] INFO     [weaver.datatype.Job] 00:00:04  53% running    [2022-08-18 20:44:01] INFO     [weaver.processes.wps_package|ncdump]  20% running    [provider: https://pavics.ouranos.ca/twitcher/ows/proxy/hummingbird, step: ncdump] - Preparing inputs/outputs for remote execution.\n",
```

Comment:
Is there a regex to escape those `XXX% running` values?  I guess they will be different for each run.

Alternatively you can put the whole cell under `# NBVAL_IGNORE_OUTPUT`.

Reply:
Use `# NBVAL_IGNORE_OUTPUT` only if you do not need to validate the format or the content of the output, otherwise, the regex escape is a safer route.

Reply:
`NBVAL_IGNORE_OUTPUT` is used because the specific logs are not that necessary to match exactly and are prone to a lot of minor changes. They are displayed mostly for demonstration in the docs.